dbot/dbot/CommandModules/NominationsModule.cs

```diff
@@ -3,25 +3,29 @@
 using System.Text;
 using System.Threading.Tasks;
 using dbot.Services;
+using dbot.Models;
 using Discord;
 using Discord.Commands;
 using Discord.WebSocket;
+using dbot.Data;
 
 namespace dbot.CommandModules
 {
     [Group("Nominate")]
     [Summary("Commands for nominating movies")]
     public class NominationsModule : ModuleBase
     {
-        private readonly NominationsService _nominationsService;
-        private readonly OmdbService _omdbService;
-        private readonly VotingService _votingService;
+        private NominationsService _nominationsService;
```

Comment:
Is there a change that required removing readonly from these?

Reply:
Not sure why this one is a conversation and the rest are reviews lol

Reply:
Yes - NominationsService and VotingService now have a dependency on DbContext. I want to keep DbContext with a scoped lifetime and thought that should also apply to NominationsService and VotingService due to this dependency. OmdbService wasn't changed so  i can add readonly back to it. The question that arises is if NominationsService and VotingService are readonly , implying their singleton lifetime, if their DbContext property with a scoped lifetime will continue to work as expected.